Source/GoBot/GoBot.swift

```diff
@@ -955,7 +967,7 @@ class GoBot: Bot {
             return
         }
 
-        utilityQueue.async {
+        userInitiatedQueue.async {
```

Comment:
Was this change necessary? Or just a test?

Reply:
It was intentional but not something directly related to this PR. I just noticed that adding blobs was happening on the utility queue, which should be reserved for actions that the user isn't waiting on. But when we are storing blobs the user is waiting on that so I changed it to the userInitiatedQueue.